terraform/modules/network-validator/scripts/network_validation.sh

```diff
@@ -802,6 +823,21 @@ then
   then
     echo "WARNING: Exposing the WebLogic administrator port [${ADMIN_HTTPS_PORT}] in the subnet [{$WLS_SUBNET_OCID}] to the internet [${ALL_IPS}] allows any user to access the WebLogic console, which is not a recommended practice. Ensure that only a specific CIDR range can access the WebLogic console. ${NETWORK_VALIDATION_MSG}"
   fi
+
+  # Check if Admin Console HTTPS Port is open for access by WLS subnet CIDR for secure mode
```

Comment:
Please change "Admin Console HTTPS Port" to "Administration Port". This is the name of the port in WLS documentation so we should keep the name consistent.